browser/tracer.all.js

```diff
@@ -2,7 +2,7 @@
 
 import opentelemetry from '@opentelemetry/api';
 // import { UserInteractionPlugin } from '@opentelemetry/plugin-user-interaction';
-import { XMLHttpRequestPlugin } from '@opentelemetry/plugin-xml-http-request';
+// import { XMLHttpRequestPlugin } from '@opentelemetry/plugin-xml-http-request';
```

Comment:
why not delete these lines?

Reply:
I left it to be able to uncomment it easily and give it try either separately or together